app/controllers/availabilities_controller.rb

```diff
@@ -43,11 +67,7 @@ def create_missing_availabilities
     end
   end
 
-  def start_time
-    @time_ranges.first.start_time.beginning_of_hour
-  end
-
-  def end_time
-    (@time_ranges.last.end_time - 1).beginning_of_hour + 1.hour
+  def check_admin
```

Comment:
Use `load_and_authorize_resource` on top

Reply:
I was just using what was in `user_controller.rb`. Should I change that as well?

Reply:
@donjar bump

Reply:
Yes just change that. Is that in this branch too?

Reply:
It was merged already in master

Reply:
Oh ok just change it in this branch as well

Reply:
Done